app/views/pages/api.html.erb

```diff
@@ -11,9 +11,8 @@
       ut labore et dolore magna aliqua. Ut enim ad minim veniam,
       quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
 
-      Besök <a href="#">dokumentationen</a> för att veta mer!
+      Besök <a href="#">dokumentationen</a> för att veta mer!                                               <!-- TODO -->
```

Comment:
Good, but not what i expected. A todo should always describe what you expect the next person, or yourself in the future to do.

Then we always know later what is expected and if no-one removes the TODO after fixing the issue we also know when it's outdated.

Example.

```html
<!-- TODO: Fill in the `href` with the correct URL when that page exists. -->
Besök <a href="#" ...
```